ocaml/xapi/storage_smapiv1_wrapper.ml

```diff
@@ -453,11 +472,12 @@ functor
         in
         List.fold_left perform_one vdi_t ops
 
-      let perform_nolock context ~dbg ~dp ~sr ~vdi ~vm this_op =
+      let perform_nolock context ~dbg ~name ~dp ~sr ~vdi ~vm this_op =
```

Comment:
I think I see what your question was @jameshensmancitrix about this function missing in Storage_mux, and I think missing this function is fine, since it is called by the higher level epoch_begin/etc. functions below, the perform function is just a shared implementation of the API, but not exposed on the surface level of the API.

Reply:
yeah add tracing just for the public APIs that exist in Storage_mux. Is there anything else apart from `perform_nolock` that is missing from Storage_mux?

Reply:
No